wpiutil/generate_numbers.py

```diff
@@ -28,10 +28,16 @@
     if os.path.exists(outputPath):
         os.remove(outputPath)
 
+    importsString = ""
+
     for i in range(MAX_NUM + 1):
+        importsString += "import edu.wpi.first.wpiutil.math.numbers.N"
```

Comment:
This would be cleaner as an f-string if we're ok with upgrading to python 3.6. It would look like `importsString += f"import edu.wpi.first.wpiutil.math.numbers.N{i};\n"`

Reply:
If we do that, styleguide complains about the string being to long, and moves things to a new line weirdly.

Reply:
That's yapf being bad. Black does a lot better, but it won't be google style anymore. I'm fine with that, to be honest.

Reply:
That f-string append line is only 78 characters, which shouldn't get wrapped.

Reply:
If it does wrap, it gave me an indentation error. I'm not comfortable enough in python, so if we want to do updates like that I think it should be in a separate pr. So we can at least fix the builds.

Reply:
Indentation error from what exactly?

Reply:
I had it before as an extra +, and when it moved that to the next line I got an indentation error and didn't know how to fix it.

Reply:
Also updating the python version might not be easily possible because we use what's default on 18.04. So updating isn't always possible

Reply:
18.04 seems to have python 3.7.

Reply:
Hmm then python 3.7 should already be installed.

Reply:
Ok. python 3.7 is installed on the docker image anyway, so we're good there.